src/account/credentials.cpp

```
@@ -0,0 +1,633 @@
+
+std::string HomeDirectory() {
+#if defined(_WIN32)
+    const std::string local = Env("LOCALAPPDATA");
+    if (!local.empty()) {
+        return local;
+    }
+    return Env("USERPROFILE");
+#else
+    const std::string home = Env("HOME");
+    if (!home.empty()) {
+        return home;
```

Clang (14.0.6)

[error] constness of 'home' prevents automatic move

(performance-no-automatic-move,-warnings-as-errors)